public/admin/config.yml

```diff
@@ -718,6 +718,11 @@ collections:
             widget: "string"
             required: false
             hint: This is the title for the Events section header. For example, "Main Stage Events." Defaults to 'Events.'
+          - name: "kicker"
+            label: "Section Kicker"
+            widget: "string"
+            required: false
+            hint: "This is the subtitle text that appears above the section title. Defaults to 'Explore'."
```

Comment:
Subtitle isn't correct: `This is the text that appears above the main section title.`